### PR TITLE
Improve the wording

### PR DESCRIPTION
**Pull request by reddymahendra52**
Tried to improve the wording.
In reference to issue #7055

### CODE REVIEW

File: files/en-us/learn/css/building_blocks/values_and_units/index.html [hunk 1]
```diff
@@ -300,7 +300,7 @@ <h3 id="RGB_and_RGBA_values">RGB and RGBA values</h3>
...
+<p><strong>Note</strong>: At some point the specifications were updated, as of CSS Colors Level 4, <code>rgba()</code> is an alias for <code>rgb()</code> and <code>hsla()</code> is an alias for <code>hsl()</code> (see below).In browsers that implement the Level 4 standard, they accept the same parameters and behave the same way. So for example both <code>rgba()</code> and <code>rgb()</code> accept colors with and without alpha channel values. Try changing the above example's <code>rgba()</code> functions to <code>rgb()</code> and see if the colors still work! Which style you use is up to you, but separating out non-transparent and transparent color definitions to use the different functions gives (very) slightly better browser support and can act as a visual indicator of where transparent colors are being defined in your code.</p>
```
Comment (rachelandrew) [inline files/en-us/learn/css/building_blocks/values_and_units/index.html:303]:
```suggestion
<h4>note:</h4>
<p>As of CSS Colors Level 4, <code>rgba()</code> is an alias for <code>rgb()</code>, and <code>hsla()</code> is an alias for <code>hsl()</code> (see below). In browsers that implement the Level 4 standard, they accept the same parameters and behave in the same way. So for example both <code>rgba()</code> and <code>rgb()</code> accept colors with and without alpha channel values. Try changing the above example's <code>rgba()</code> functions to <code>rgb()</code> and see if the colors still work! Which style you use is up to you, but separating out non-transparent and transparent color definitions to use the different functions gives (very) slightly better browser support and can act as a visual indicator of where transparent colors are being defined in your code.</p>
```

**Review by rachelandrew — CHANGES_REQUESTED** [review]
Please see my suggested update inline.

File: files/en-us/learn/css/building_blocks/values_and_units/index.html (hunk 1 above)
Comment (reddymahendra52) [reply files/en-us/learn/css/building_blocks/values_and_units/index.html]: Requested changes have been done.
Can you please check??

**Comment by reddymahendra52** [comment]
Requested changes have been done.

File: files/en-us/learn/css/building_blocks/values_and_units/index.html (hunk 1 above)
Comment (ConsciousClay) [reply files/en-us/learn/css/building_blocks/values_and_units/index.html]: It's not clear what the significance is of "the same parameters" and "behave in the same way". If the browsers implement a standard, then they implement the standard equally, right? Isn't that just what it means to implement a standard? Besides, beginners don't know what is meant by "CSS colors level 4". This kind of language hasn't been used in these tutorials heretofore.

This sentence, "So for example both rgba() and rgb() accept colors with and without alpha channel values." needs to say **or**, not **and**. A function like rgb() cannot accept a color with AND without an alpha channel. Conjunctions don't work like that in English. We have to say "So, for example, both rgba() and rgb() accept colors with OR without alpha channel values." Note the commas.

This clause needs to be reworked: "separating out non-transparent and transparent color definitions to use the different functions". We separate definitions to use functions? I can't make sense of that. We don't separate definitions at all. If I define a word, and then I define another word, those two definitions are _already_ separate. Definitions are not things we can separate. And I don't understand the purpose of using functions as stemming from our separation of definitions.

**Review by rachelandrew — APPROVED** [review]
thanks :)

**Pull request merged**